base/uk.ac.stfc.isis.ibex.opis/resources/RIKEN/individual_kicker_row.opi

```diff
@@ -102,7 +97,7 @@ $(pv_value)</tooltip>
     <actions hook="false" hook_all="false" />
     <alarm_pulsing>false</alarm_pulsing>
     <auto_size>false</auto_size>
-    <backcolor_alarm_sensitive>false</backcolor_alarm_sensitive>
+    <backcolor_alarm_sensitive>true</backcolor_alarm_sensitive>
```

Comment:
Using 2 seperate mechanisims for setting background colours seems to cause problems (Colours are different depending on order of change between states, colours dont load until opi is refreshed etc.) , given the different cases here we should use rules rather than this.